security/advancedtls/testdata/crl/provider/README.md

```diff
@@ -0,0 +1,73 @@
+About This Directory
+-------------
+The directory test data (certificates and CRLs) used for testing CRL providers 
+functionality.
+
+How to Generate Test Data Using OpenSSL
```

Comment:
Optional: it's nice that you show how to regenerate everything, but it would be even nicer if there was just a script that could be run that does it instead.